src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/policy/ResidentialChecksPolicyV1.kt

```diff
@@ -3,22 +3,22 @@ package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.polic
 import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.residentialchecks.Input
 import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.residentialchecks.InputType
 import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.residentialchecks.Option
-import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.residentialchecks.Question
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.residentialchecks.PolicyVersion
 import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.residentialchecks.ResidentialChecksPolicy
-import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.residentialchecks.ResidentialChecksPolicyVersion
-import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.residentialchecks.ResidentialChecksTask
 import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.residentialchecks.Section
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.residentialchecks.Task
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.residentialchecks.TaskQuestion
 
 val RESIDENTIAL_CHECKS_POLICY_V1 = ResidentialChecksPolicy(
-  version = ResidentialChecksPolicyVersion.V1,
+  version = PolicyVersion.V1,
   tasks = listOf(
-    ResidentialChecksTask(
+    Task(
       code = "address-details-and-informed-consent",
       name = "Address details and informed consent",
       sections = listOf(
         Section(
           questions = listOf(
-            Question(
+            TaskQuestion(
```

Comment:
Had to rename this as there was already a Question class for the eligibility and suitability checks and this overwriting that one in the type def file for the front end. I don't know if there's a way around this?

Reply:
oh yes - this is a really long outstanding issue with openapi, I think it might be possible to change it to use a fully qualified name but then it'd be quite difficult to read